plugins/kotlin/j2k/shared/tests/testData/newJ2k/postProcessing/GuardClause.kt

```diff
@@ -1,5 +1,52 @@
 class Test {
-    fun test(s: String?) {
-        requireNotNull(s) { "s should not be null" }
+    fun testRequire(s1: String, b1: Boolean, b2: Boolean) {
+        requireNotNull(s1) { "s should not be null" }
```

Comment:
This is now redundant, because `s` has a non-null type. I met the same issue during my experiments with nullability, so I will probably fix it later (the whole `requireNotNull` call can be removed).